cabal.project

```diff
@@ -30,6 +30,8 @@ package bittide
   -- so we're able to import its css file from the custom theme.
   haddock-options: --theme=linuwial-wrap-types.css --theme=Linuwial
 
+  -- flags: -rusty-callisto
+
```

Comment:
Was this added by accident?
For development purposes you can create a `cabal.project.local`